content/en/training/_index.html

```diff
@@ -88,7 +88,7 @@ <h5>
         </h5>
         <p>The Certified Kubernetes Administrator (CKA) program provides assurance that CKAs have the skills, knowledge, and competency to perform the responsibilities of Kubernetes administrators.</p>
         <br>
-        <a href="https://training.linuxfoundation.org/certification/certified-kubernetes-application-developer-ckad/" target="_blank" class="button">Go to Certification</a>
+        <a href=https://training.linuxfoundation.org/certification/certified-kubernetes-administrator-cka/" target="_blank" class="button">Go to Certification</a>
```

Comment:
@celestehorgan , double check that the `href` attribute is quoted correctly (here and other links on the page).

Reply:
I caught this on a final test. Thanks though!